src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -688,6 +691,40 @@ To activate this behavior set `pa.ldap.group.membership.fallback` to true, in LD
 * If a user is not found in LDAP, fall back to authentication and group membership checking with login
 and group files. To activate this behavior, set `pa.ldap.authentication.fallback` to true, in LDAP configuration file.
 
+=== PAM
+
+The ProActive Scheduler & Resource Manager are able to interact with *PAM (Linux Pluggable Authentication Modules)* to check users login/password.
+It is not currently possible to retrieve linux system group memberships for users. Thus groups must be managed using the `PROACTIVE_HOME/config/authentication/group.cfg` file.
+
+The *first step* to authenticate users with PAM is to configure the PAM login methods in the scheduler and rm settings:
```

Comment:
I think it would be more readable to have an enumeration list (with a dedicated paragraph) or subsections for each step.